sympy/crypto/crypto.py

```diff
@@ -256,8 +256,27 @@ def encipher_substitution(pt, key, symbols="ABCDEFGHIJKLMNOPQRSTUVWXYZ"):
 #################### Vigenère cipher examples ########################
 ######################################################################
 
+def keystring(phrase, repeats=True):
+    """
+    Returns a keystring formatted in a way as needed by Vigenère encryption.
+
+    Examples
+    ========
+
+    >>> keystring('with repetitions')
+    'WITHREPETITIONS'
+
+    >>> keystring('without repetitions', False)
+    'WITHOUREPNS'
 
-def encipher_vigenere(pt, key, symbols="ABCDEFGHIJKLMNOPQRSTUVWXYZ"):
+    """
+    rv = ''.join(''.join([x for x in phrase.split() if x.isalnum()]).upper())
+    if repeats:
+        return rv
+    return ''.join(uniq(rv))
+
+
+def encipher_vigenere(pt, key, repeats = True, symbols="ABCDEFGHIJKLMNOPQRSTUVWXYZ"):
```

Comment:
Unless you are intentionally trying to break compatibility (which, if repeats=True, might not be a bad idea) since `symbols`  was formerly in position 3, it should stay there and the new positional argument should be last.